src/fabric/ScreenNativeComponent.ts

```diff
@@ -21,6 +21,10 @@ type TransitionProgressEvent = Readonly<{
   goingForward: Int32;
 }>;
 
+type HeaderChangeEvent = Readonly<{
```

Comment:
`HeaderHeightChange`?

Reply:
Renamed in https://github.com/software-mansion/react-native-screens/pull/1802/commits/3c9c0fbe13cb07071e71254d249b2f90a964c3a0 👍